content/troubleshooting/common-ios-issues.md

```diff
@@ -226,3 +226,22 @@ Apple has updated Apple Developer Program License Agreement and it needs to be r
 
 ###### Solution
 In order to update your existing apps and submit new apps to the App Store, the Account Holder must review and accept the updated agreement by signing in to their [account](https://appstoreconnect.apple.com/agreements/#/) on the Apple Developer website.
+
+### Certificate will no longer be valid in 30 days
+
+###### Description
+You received e-mail from Apple stating that:
+
+    Your Distribution Certificate will no longer be valid in 30 days. To generate a new certificate, sign in and visit Certificates, Identifiers & Profiles.
+
+###### Cause
+Distribution certificates issued by Apple have a validity period of one year from the date of issuance. After one year, they expire, which means they can no longer be used to sign and distribute apps.
+
+###### Solution
+Rest assured, Codemagic simplifies the certificate management process for you. Here's how it works:
+ - If you're using `automatic code signing`, Codemagic takes care of certificate and provisioning profile management for you. This means it also automatically renews the certificate when it's about to expire.
```

Comment:
Let's explain it sth like this way:

_By providing api key, issuer id etc. Codemagic can take care of........_